ports/duckdb/vcpkg.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "duckdb",
+  "version": "0.9.1",
+  "description": "High-performance in-process analytical database system",
+  "homepage": "https://duckdb.org",
+  "license": "MIT",
+  "supports": "!(uwp | android | (windows & arm64))",
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ],
+  "default-features": [
+    "parquet"
+  ],
+  "features": {
+    "autocomplete": {
+      "description": "Statically link the autocomplete extension into DuckDB"
+    },
+    "fts": {
+      "description": "Statically link the FTS extension into DuckDB"
+    },
+    "httpfs": {
+      "description": "Statically link the httpfs extension into DuckDB"
+    },
```

```suggestion
    "httpfs": {
      "description": "Statically link the httpfs extension into DuckDB",
      "dependencies": [
        "openssl"
      ]
    },
```

The related build error:
```
CMake Error at F:/Program Files/CMake/share/cmake-3.27/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the
  system variable OPENSSL_ROOT_DIR (missing: OPENSSL_CRYPTO_LIBRARY
  OPENSSL_INCLUDE_DIR)
Call Stack (most recent call first):
  F:/Program Files/CMake/share/cmake-3.27/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  F:/Program Files/CMake/share/cmake-3.27/Modules/FindOpenSSL.cmake:668 (find_package_handle_standard_args)
  F:/duck/scripts/buildsystems/vcpkg.cmake:855 (_find_package)
  extension/httpfs/CMakeLists.txt:19 (find_package)
```